src/journal-remote/journal-remote.c

```diff
@@ -727,15 +731,23 @@ static int setup_microhttpd_server(RemoteServer *s,
                 goto error;
         }
 
-        r = sd_event_add_io(s->events, &d->event,
+        r = sd_event_add_io(s->events, &d->io_event,
                             epoll_fd, EPOLLIN,
                             dispatch_http_event, d);
         if (r < 0) {
                 log_error_errno(r, "Failed to add event callback: %m");
                 goto error;
         }
 
-        r = sd_event_source_set_description(d->event, "epoll-fd");
+        r = sd_event_add_time(s->events, &d->timer_event,
+                              CLOCK_BOOTTIME, ULONG_LONG_MAX, 5000,
+                              null_timer_event_handler, d);
+        if (r < 0) {
+                log_error_errno(r, "Failed to add timer_event: %m");
+                goto error;
+        }
+
+        r = sd_event_source_set_description(d->io_event, "epoll-fd");
```

Comment:
btw, it would be nice to set precise description on the io and time events, i.e. change the io-one to "io-event", and the timer one to "time-event".

setting a description is purely a debugging thing, but we try to set them on all events we allocate

Reply:
Ohh yeah, saw that and will add it.